src/config/dynamic-theme-fixes.config

```diff
@@ -41,6 +41,19 @@ div[class^='scrollableList'] {
 
 ================================
 
+itslearning.com
+
+INVERT
+.ccl-feedback-tip
+.c-dropdownMenu__navigation
+
+CSS
+#ctl00_PageBody {
+    background-color: var(--darkreader-neutral-background);
```

Comment:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

Reply:
Important isn't required here

Reply:
It's standard practice to add `!important;` where possible.